project_templates/square_pypi_package/{{cookiecutter.name}}/CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 # Change log
 
-<!-- Format for headings: 1.2.3 (YYYY-MM-DD) -->
+All notable changes to {{cookiecutter.name}} will be documented in this file.
 
-## Unreleased
+Versioning follows [semver](https://semver.org/).
 
-- First release of {{cookiecutter.name}}
+Dependencies are updated to the latest available version during each release. Those changes are not noted here explicitly.
+
+This project uses [scriv](https://scriv.readthedocs.io/en/stable/) to maintain the change log.
+Changes for the upcoming release can be found in [changelog.d](https://github.com/{{cookiecutter.github_org}}/{{cookiecutter.name}}/tree/main/changelog.d/).
```

Comment:
Many of the same thoughts here as for the FastAPI template.

The one thing I'm not sure about is saying we don't document dependency changes here. I think that dependency constraint changes for PyPI packages probably should be mentioned in the changelog since they impact downstream usage. I also don't think that "Dependencies are updated to the latest available version during each release." applies to our PyPI packages.